apps/builder/app/env/env.server.ts

```diff
@@ -77,6 +77,14 @@ const env = {
    */
   ENTRI_APPLICATION_ID: process.env.ENTRI_APPLICATION_ID ?? "webstudio",
   ENTRI_SECRET: process.env.ENTRI_SECRET,
+
+  /**
+   * Projects as templates in dashboard
+   */
+  PROJECT_TEMPLATES:
+    process.env.PROJECT_TEMPLATES?.split(",").map((projectId) =>
+      projectId.trim()
+    ) ?? [],
```

Comment:
.split(/\s*,\s*/) ?? []

Reply:
Will not solve the edgecases when 1st or last item has spaces at begin and end.